test/fixtures/rules/css-modules/undefined.js

```diff
@@ -0,0 +1,15 @@
+// undefined.js
```

Comment:
You may remove this comment. I added it just for you to know that it was the `undefined.js` file.

Reply:
removed